specification/socks5plus/src/main/scripts/org/kaazing/specification/socks5plus/request/bind/connection.request.bind.with.invalid.uri.request.rpt

```diff
@@ -0,0 +1,69 @@
+connect tcp://localhost:1080
+connected
+
+## Write the request bytes for connecting with no-authentication
+# version: SOCKS v5
+write [0x05]
+
+# number of method: 1
+write [0x01]
+
+# method: no authentication required
+write [0x00]
+
+
+## Read the response, expect success
+# version: SOCKS v5
+read [0x05]
+
+# selected method: no authentication required
+read [0x00]
+
+## Write BIND request
+# version: SOCKS v5
+write [0x05]
+
+# BIND command
+write [0x02]
+
+# RSV field
+write [0x00]
+
+# ATYP with URI
+write [0xFE]
+
+# DST.ADDR with invalid URI
+# URI length
+write [0x00 0x00 0x00 0x09]
+
+# Invalid URI
+write "abc://foo" [0x00]
```

Comment:
What's the end 0x00 (again, I see that gateway doing this way) ? If the length of address is 9, here we are sending 10 bytes.